### PR TITLE
fix: fix readme so new users can run the project successfully

### PR DESCRIPTION
**Pull request by ctindogaru**
Following the current instructions inside the README lead to several errors.

Make sure these do not occur anymore.

### CODE REVIEW

**Review by mikedotexe — DISMISSED** [review]
Great work, thanks. Very necessary to merge this

File: .gitignore [hunk 1]
```diff
@@ -27,7 +27,6 @@ yarn-debug.log*
 yarn-error.log*
 
 # local env files
-.env.local
```
Comment (AlexeyPonomarenko) [inline .gitignore]: Please, do not remove this file from gitignor.

File: .env.local [hunk 2]
```diff
@@ -0,0 +1,4 @@
+NEXT_PUBLIC_API_URL=https://api.dev.app.astrodao.com
```
Comment (AlexeyPonomarenko) [inline .env.local]: This file should not be present in the repo. Please remove this. You can provide proper description in README.md if you feel it's necessary.

**Review by AlexeyPonomarenko — CHANGES_REQUESTED** [review]

File: .env.local (hunk 2 above)
Comment (mikedotexe) [reply .env.local]: Is this a sensitive URL?

Comment (mikedotexe) [reply .env.local]: And yes, it's absolutely necessary to have a working readme.

File: .gitignore (hunk 1 above)
Comment (TrevorJTClarke) [reply .gitignore]: Another pattern that can be done is `.env.example`, which can then be copied with npm postinstall to `.env.local`, so `.env.example` comes pre-loaded with defaults and sample strings

File: .env.local (hunk 2 above)
Comment (dmytro-almighty) [reply .env.local]: It's not sensitive, just .env.local should not be part of the project. Moving variables to .env. an example is a better idea :)

**Comment by dmytro-almighty** [comment]
we implemented it in separate PR, it already merged

**Comment by RomaSha2010** [comment]
Closed with unmerged commits

**Pull request closed without merging**